README.md

```diff
@@ -130,14 +135,17 @@ final class SimpleConsumer implements Consumer
         return ['topic1'];
     }
 
-    public function getConfig() : Config
+    /**
+     * @return Configuration
+     */
+    public function getConfiguration() : Configuration
     {
-        return new Config($this->getGroupId());
+        return new Configuration($this->getGroupId());
     }
 }
 ```
 
- It is automatically registered to container for it `implements Consumer` 
+Consumer will be register automatically to use.
```

Comment:
Why do you think this needs to be changed?

Reply:
I modified it like 2 days ago, you just was a bit faster ;). The `extends` fix was necessary for sure but overall the previous formulation sounds better to me.